packages/enterprise/src/license.ts

```diff
@@ -189,16 +191,6 @@ export async function getVerifiedLicenseData(key: string) {
   }
   delete decodedLicense.eat;
 
-  // The `trial` field used to be optional, force it to always be defined
-  decodedLicense.trial = !!decodedLicense.trial;
```

Comment:
Something else might still rely on this code.  Better to leave it in.

Reply:
Good call, I'll leave that in.